client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -35,6 +36,11 @@ const StatsInsights = ( props ) => {
 	// Necessary to properly configure the fixed navigation headers.
 	sessionStorage.setItem( 'jp-stats-last-tab', 'insights' );
 
+	const lastYear = moment.utc().subtract( 1, 'year' ).format( 'YYYY-MM-DD' );
+	const isPrevArrowHidden = ( queryDate ) => {
+		return moment( queryDate ).isSameOrBefore( '2001-01-01' );
```

Comment:
Should I limit this to 2000 or allow any year?

Reply:
I second Filipe's recommendation on "[disabling] previous years without data"!